### DISCUSSION TITLE
How can I make a glow effect？

### DISCUSSION THREAD

**Discussion by dabing1022** [keiwando/toonsquid#149] [Q&A]
I don't know how to make a glow effect by toonsquid？ Any ideas？

**Comment by DinoAwakening** [comment]
You can use a blurry brush at 1% flow, and make strokes over ando over until it gets light alike xd

**Reply by dabing1022** [reply]
I will try this way. Thanks☺️

By the way，wish toonsquid add a feature called filter. We can add glow，blur effects.

**Reply by DinoAwakening** [reply]
SAMEEE. I’d love to use motion blur and gaussian blur x-x

**Reply by DJ1KANM8Z** [reply]
as someone that loves glow i need this ASAP i love adding a glow effect to my work, like in flipaclip. although im moving to this app so i hope a glow effect can become a thing soon.
